.gitattributes

```diff
@@ -0,0 +1 @@
+yarn.lock linguist-generated=true
```

Comment:
`yarn.lock` is generated. We're not interested in reviewing it, nor in having it count towards our lines of code in anyway.